### PR TITLE
Closes #1 Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Online Shop Vydelka</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (BohdanRomaniuk) [inline index.html]: You need to add new line

Comment (Bozhen9ka) [inline index.html]: Need to add new line

Comment (dimkijeee) [inline index.html]: Add new line, please

Comment (mshastun97) [inline index.html]: Add new line!

Comment (VitalikKoval) [inline index.html]: Add new line.

Comment (MariiaHapatyn) [inline index.html]: Add new line.

Comment (stanislavsvystovych) [inline index.html]: Need to add new line

Comment (romank97l) [inline index.html]: Add new line

**Review by dimkijeee — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (MariiaHapatyn) [inline index.html]: Add new line.

**Review by stanislavsvystovych — CHANGES_REQUESTED** [review]

**Review by BohdanRomaniuk — CHANGES_REQUESTED** [review]
You need to add new line

**Review by mshastun97 — CHANGES_REQUESTED** [review]

**Review by VitalikKoval — CHANGES_REQUESTED** [review]

**Review by romank97l — CHANGES_REQUESTED** [review]

**Review by VitalikKoval — APPROVED** [review]

**Review by BohdanRomaniuk — APPROVED** [review]

**Review by romank97l — APPROVED** [review]

**Review by dimkijeee — APPROVED** [review]

**Review by mshastun97 — APPROVED** [review]

**Review by MariiaHapatyn — APPROVED** [review]

**Review by stanislavsvystovych — APPROVED** [review]

**Pull request merged**